.github/workflows/build-avr-gcc.yml

```diff
@@ -0,0 +1,68 @@
+name: Build AVR GCC
+
+on:
+  push:
+  workflow_dispatch:
+
+jobs:
+  build-avr-gcc:
+    runs-on: ubuntu-20.04
+
+    steps:
+      - name: Install dependencies
+        run: |
+         sudo apt-get update
+         sudo apt-get upgrade -y
+         sudo apt-get install -y git wget autoconf2.64 build-essential netpbm libmpc-dev libmpfr-dev libgmp-dev libmpfr6 texinfo doxygen flex bison libexpat1-dev slang-xfig
+
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Build
+        run: |
+         ls
+         bash ./build.sh
+
+      - name: Compiling test files
+        run: |
+         /opt/avr-gcc/bin/avr-g++ --version
+         /opt/avr-gcc/bin/avr-g++ -mmcu=atmega328p test/main1.cpp
+         /opt/avr-gcc/bin/avr-g++ -mmcu=atmega328p test/main2.cpp
+
+      #- name: Fake toolchain
+      #  run: |
+      #   mkdir -p /opt/avr-gcc/bin
+      #   cp $(which true) /opt/avr-gcc/bin/avr-g++
+
```

Comment:
Should these lines be removed?

Reply:
I intentially left the lines there, might be useful in the future if CI requires debugging...

Reply:
Then we leave it :+1: